runner/src/server/plugins/engine/pageControllers/PlaybackUploadPageController.ts

```diff
@@ -0,0 +1,102 @@
+import { PageController } from "server/plugins/engine/pageControllers/PageController";
+import { FormModel } from "server/plugins/engine/models";
+import { Page } from "@xgovformbuilder/model";
+import { FormComponent } from "server/plugins/engine/components";
+import { HapiRequest, HapiResponseToolkit } from "server/types";
+
+export class PlaybackUploadPageController extends PageController {
+  inputComponent: FormComponent;
+
+  constructor(model: FormModel, pageDef: Page, inputComponent: FormComponent) {
+    super(model, pageDef);
+    this.inputComponent = inputComponent;
+  }
+
+  buildRadioViewModel(error?: string) {
+    const standardViewModel = {
```

Comment:
I'd probably set this to the class itself, otherwise it's going to create this object every time buildRadioViewModel is called